news/2022-02-15-beatmap-spotlights-season-5-winter-2022.md

```diff
@@ -0,0 +1,162 @@
+
+## The changes
+
+### New spotlights playlists format
+
+We've heard a lot of voices that the old playlists format might've been a little too boring for more experienced players as one or two extras and/or insanes per list is just simply not enough to be interesting. To increase the competition, and at the same time, to keep it also approachable for newer players, we decided to reshuffle how the playlists system work. **From now on, each playlist will last 2 weeks and will include 2 hard, 3 insane, and 4 extra difficulties.** Each season will consist of 3 playlists, after the first run, the playlists will be re-played for a week.
+
+### Themed playlists
+
+But that's not all regarding the playlists changes. **Starting this season, the first playlist of every season will be themed with some kind of gimmick.** We've came to a conclusion that spotlights playlists might've felt a little disconnected, to make them more interesting, we decided to make the first playlist of every incoming season special. **This time, the theme is unorthodox rhythm of any sorts**.
+
+### More transparency
+
+There's been a lot of voices that nobody really knows who's the member and what's going on in the Spotlighs Team. We'd like to face this issue, and make it so everything is a little bit more clear to the outsiders. Each season announcement will now include *[Inside the spotlights team](inside-the-team)* section, listing the most recent team changes. We will also credit respective curators, resposnible for suggesting the maps, in the [picks lists](playlists). There's more to come in that regard, so stay tuned for the next seasons!
+
+## Medals and beatmap packs
+
+Many people stated that they'd like to enjoy some kind of spotlights experience on stable, like it used to be in the past. To make it somehow possible, we will now release beatmap packs along with medal set for each gamemode once all of the playlists are released (in 4 weeks). We also have plans for releasing medal packs for previous Spotlights seasons. Have fun jumping on grindset to catch them all!
+
+## <a id="inside-the-team"></a> Inside the spotlights team
+
+In this section we will cover member changes and important happenings inside the Spotlights Team.
+
+**Curator additions**
+- **osu!taiko**: Hivie, radar
+- **osu!catch**: Rocma
+- **osu!mania**: \[ Decku \], Kibitz
+
+**Curator resignments**
+- **osu!**: Icekalt, xbopost
+- **osu!taiko**: Charlotte
+- **osu!catch**: Deif, JBHyperion, Spectator
+- **osu!mania**: Raveille
+
+In addition to that, pishifat will leave the management. After a long service, Hivie will replace him as a helping hand in making the things work as intended.
+
+For the full Spotlights Team members list, check out the [Beatmap Spotlights wiki article](/wiki/Beatmap_Spotlights).
+
+## <a id="how-to-participate"></a> How to participate in Beatmap Spotlights League?

```

Comment:
league? it's only mentioned here

the header isn't worded like a question, so it can't have a question mark

the anchor isn't referenced anywhere, and also isn't really required, since headings up to level 3 automatically get an anchor. if you're linking elsewhere, you'd do `news-post-link#how-to-participate-in-the-beatmap-spotlights`

```suggestion
## How to participate in the Beatmap Spotlights
```